tests/test.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { test, expect } from '@playwright/test';
+test('my first test', async ({ page }) => {
+  
+  test.skip(!!process.env.CI, 'Test file is skipped in CI due to Cloudflare protection.');
+  await page.goto('https://practicesoftwaretesting.com/auth/login');
+  await page.locator('id=email').fill('customer@practicesoftwaretesting.com');
+  await page.locator('id=password').fill('welcome01');
+  await page.locator('data-test=login-submit').click();
```

Comment:
Consider adding testIdAttribute to your config, that way, you will not have to use locator() with CSS selector, but - getByTestId method instead - https://playwright.dev/docs/api/class-testoptions#test-options-test-id-attribute

```suggestion
  await page.getByTestId('login-submit').click();
```
And let's go through all similar locators in your test and change them accordingly